alacritty/src/input.rs

```diff
@@ -246,6 +246,55 @@ impl<T: EventListener> Execute<T> for Action {
             Action::Mouse(MouseAction::ExpandSelection) => ctx.expand_selection(),
             Action::SearchForward => ctx.start_search(Direction::Right),
             Action::SearchBackward => ctx.start_search(Direction::Left),
+            Action::NextPrompt => {
+                let terminal = ctx.terminal_mut();
+
+                let point = if terminal.mode().contains(TermMode::VI) {
+                    terminal.vi_mode_cursor.point
+                } else {
+                    let display_offset = terminal.grid().display_offset() as i32;
+                    // If we're outside of Vi mode use top most visible line.
+                    Point::new(Line(terminal.screen_lines() as i32 - 1 - display_offset), Column(0))
```

Comment:
This is not the topmost line though, is it? With a display offset of 0, the topmost line would be line `0`. With your logic and a viewport with 30 lines it would be `30 - 1 - 0 = 29`. Which is the bottommost line.

Reply:
Yeah, the comment is wrong. I need bottom most line.